src/main/java/org/mule/extension/aggregator/internal/privileged/executor/AbstractAggregatorExecutor.java

```diff
@@ -213,6 +222,26 @@ public void start() throws MuleException {
     }
   }
 
+  private void executeDeferActions() {
+    try {
+      deferredActions();
+      schedulerQuorum.stop();
+    } catch (Exception e) {
+      if (e.getClass().getName().contains(QUORUM_EXCEPTION)) {
+        LOGGER.warn("Quorum constraint not satisfied");
```

Comment:
What about "The required quorum was not reached. Waiting for quorum...", to express that we are looping until achieve the quorum.

Reply:
We modified this message for the suggestion.